addons/l10n_mx/views/report_templates.xml

```diff
@@ -0,0 +1,20 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+    <template id="l10n_mx_external_layout_standard" inherit_id="web.external_layout_standard">
+        <div t-field="company.partner_id" position="attributes">
+            <attribute t-if="company.country_id.code == 'MX'" name="t-options">{"widget": "contact", "fields": ["address", "name", "vat"], "no_marker": true}</attribute>
+        </div>
+    </template>
+
+    <template id="l10n_mx_external_layout_background" inherit_id="web.external_layout_background">
+        <span t-field="company.partner_id" position="attributes">
+            <attribute t-if="company.country_id.code == 'MX'" name="t-options">{"widget": "contact", "fields": ["address", "vat"], "no_marker": true}</attribute>
+        </span>
+    </template>
+
+    <template id="l10n_mx_external_layout_boxed" inherit_id="web.external_layout_boxed">
+        <span t-field="company.partner_id" position="attributes">
+            <attribute t-if="company.country_id.code == 'MX'" name="t-options">{"widget": "contact", "fields": ["address", "name", "vat"], "no_marker": true}</attribute>
```

Comment:
Imo, it's not something to be changed on default layout but on invoice report

Reply:
@momegahed You get a point here. I totally agree with you. However, the product owners are on holidays now and I really would like to check that with them since it's something to be changed in `base`

Reply:
as a side note, we are also already doing something similar for canada localization: https://github.com/odoo/odoo/blob/16.0/addons/l10n_ca/views/report_template.xml

Reply:
This will not work for multiple localizations... The last installed module will override the t-if again and again so this will only work for a single country at a time